config/enterprise_versions.yml

```diff
@@ -85,9 +79,15 @@ components:
   elasticsearch-metrics:
     image: tigera/elasticsearch-metrics
     version: master
+  # Below components are third-party images hosted by default under quay.io/tigera. If additional
```

Comment:
Could you change this comment, it really isn't very applicable anymore since we're rehosting all the images under quay.io/tigera?  Maybe something like
```
The components below are third-party images that have been retagged under quay.io/tigera so all enterprise images come from the same repository and org.
```